docs/architecture/distributed-cloud-native-apps-containers/introduction-to-cloud-native-development/what-is-cloud-native.md

```diff
@@ -0,0 +1,355 @@
+---
+title: What is Cloud Native?
+description: Learn about the foundational pillars that provide the bedrock for cloud-native systems
+author: robvet
+ms.date: 12/14/2023
+---
+
+# What is Cloud Native?
+
+[!INCLUDE [download-alert](../includes/download-alert.md)]
+
+Stop what you're doing and ask your colleagues to define the term "Cloud Native". There's a good chance you'll get several different answers.
+
+Let's start with a simple definition:
+
+> *Cloud-native architecture and technologies are an approach to designing, constructing, and operating workloads that are built in the cloud and take full advantage of the cloud computing model.*
+
+The [Cloud Native Computing Foundation](https://www.cncf.io/) provides the [official definition](https://github.com/cncf/toc/blob/main/DEFINITION.md):
+
+> *Cloud native practices empower organizations to develop, build, and deploy workloads in computing environments (public, private, hybrid cloud) to meet their organizational needs at scale in a programmatic and repeatable manner. It is characterized by loosely coupled systems that interoperate in a manner that is secure, resilient, manageable, sustainable, and observable.*
+
+Cloud native is about *speed* and *agility*. Business systems are evolving from enabling business capabilities to tools for strategic transformation that accelerate business velocity and growth. It's imperative to get new ideas to market immediately.
+
+At the same time, business systems have also become increasingly complex with users demanding more. They expect rapid responsiveness, innovative features, and zero downtime. Performance problems, recurring errors, and the inability to move fast are no longer acceptable. Your users will visit your competitor. Cloud-native systems are designed to embrace rapid change, large scale, and resilience.
+
+Here are some companies who have implemented cloud-native techniques. Think about the speed, agility, and scalability they've achieved.
+
+| Company | Experience |
+| :-------- | :-------- |
+| [Netflix](https://www.infoq.com/news/2013/06/netflix/) | Has 600+ services in production. Deploys 100 times per day. |
```

Comment:
I wonder if we have a Microsoft example.....

Reply:
Is this something that you're looking into?